### ISSUE TITLE
[GraphQL Plugin] Allow reading fields that have in the key `.`

### ISSUE THREAD

**Issue by taras** [thefrontside/playhouse#72]
There are lots of annotations that have comma in their keys. For example `metadata.annotations.github.com/project-slug`. We need to be able to retrieve this value with `@field(at: "metadata.annotations.github.com/project-slug")`. Currently, this doesn't work because `github.com/project-slug` goesn't get read correctly.

**Comment by taras** [comment]
Apparently, this should be possible with `@field(at: "metadata.annotations.[\"github.com/project-slug\"]")` based on https://runkit.com/taras/62f90101332c620008968390 


<blockquote><div><strong><a href="https://runkit.com/taras/62f90101332c620008968390">RunKit</a></strong></div><div>RunKit notebooks are interactive javascript playgrounds connected to a complete node environment right in your browser. Every npm module pre-installed.</div></blockquote>

**Comment by wKich** [comment]
Maybe we can use `@field(path: String[])`?

**Comment by cowboyd** [comment]
> Maybe we can use `@field(path: String[])`?

Confirm that this would be an option https://runkit.com/cowboyd/6323338e8d55a30008cf651f

can we make it:

```graphql
directive @fgield(
  at: String! | [String!]!
) on FIELD_DEFINITION 
```

<blockquote><div><strong><a href="https://runkit.com/cowboyd/6323338e8d55a30008cf651f">RunKit</a></strong></div><div>RunKit notebooks are interactive javascript playgrounds connected to a complete node environment right in your browser. Every npm module pre-installed.</div></blockquote>

**Comment by wKich** [comment]
We implemented it in #112
